code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -1,145 +1,155 @@
+	amount_per_transfer_from_this = 1
+	volume = 1
+	icon_state = "dexalin"
+
+	New()
+		..()
+		reagents.add_reagent("dexalinp", 1)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/chloralhydrate
+	name = "anesthetic autoinjector"
+	desc = "An autoinjector loaded with 1 unit of Chloral Hydrate and 9 units of Sleeping Agent. Good to quickly pacify someone, for surgery of course."
+	amount_per_transfer_from_this = 10
+	volume = 10
+	icon_state = "anesthetic"
+
+	New()
+		..()
+		reagents.add_reagent("chloralhydrate", 1)
+		reagents.add_reagent("stoxin", 9)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/Dylovene
+	name = "\improper Dylovene (anti-tox) autoinjector"
+	desc = "An auto-injector loaded with 15 units of Dylovene, an anti-toxin agent useful in cases of poisoning, overdoses and toxin build-up."
+	icon_state = "dylovene"
+
+	New()
+		..()
+		reagents.add_reagent("anti_toxin", 15)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/Tramadol
+	name = "\improper Tramadol autoinjector"
+	desc = "An auto-injector loaded with 15 units of Tramadol, a weak but effective painkiller for normal wounds."
+	icon_state = "tramadol"
+
+	New()
+		..()
+		reagents.add_reagent("tramadol", 15)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/Oxycodone
+	name = "\improper Oxycodone (EXTREME PAINKILLER) autoinjector"
+	desc = "An auto-injector loaded with 5 units of Oxycodone, a powerful pankiller intended for life-threatening situations."
+	amount_per_transfer_from_this = 10
+	volume = 10
+	icon_state = "oxycodone"
+
+	New()
+		..()
+		reagents.add_reagent("oxycodone", 10)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/Kelo
+	name = "\improper Kelotane autoinjector"
+	desc = "An auto-injector loaded with 15 units of Kelotane, a common burn medicine."
+	icon_state = "kelotine"
+
+	New()
+		..()
+		reagents.add_reagent("kelotane", 15)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/Bicard
+	name = "\improper Bicaridine autoinjector"
+	desc = "An auto-injector loaded with 15 units of Bicaridine, a common brute and circulatory damage medicine."
+	icon_state = "bicaridine"
+
+	New()
+		..()
+		reagents.add_reagent("bicaridine", 15)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/Inaprovaline
+	name = "\improper Inaprovaline autoinjector"
+	desc = "An auto-injector loaded with 15 units of Inaprovaline, an emergency stabilization medicine for patients in critical condition."
+	icon_state = "clonefix" //TEMP
+
+	New()
+		..()
+		reagents.add_reagent("inaprovaline", 15)
+		update_icon()
+
+/obj/item/reagent_container/hypospray/autoinjector/hypervene
+	name = "\improper Hypervene autoinjector"
+	desc = "An auto-injector loaded with 3 units of Hypervene, an emergency medicine that rapidly purges chems, radiation and toxins. Causes pain and vomitting."
+	icon_state = "clonefix" //TEMP
+
+	New()

```

Comment:
Full path, please. That is `/obj/item/reagent_container/hypospray/autoinjector/hypervene/New()`